### PR TITLE
Added support for secondary groups

### PR DESCRIPTION
**Pull request by madelen-axis**
NB! If a container needs sub-group access it has to use the group-add flag.

### Describe your changes

Adding mapping for secondary groups to /etc/subgid. This makes it possible to add secondary groups to containers and have them mapped to real groups on the device.

### Checklist before requesting a review

- [X] I have performed a self-review of my own code
- [X] I have verified that the code builds perfectly fine on my local system
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have verified that my code follows the style already available in the repository
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -330,6 +331,31 @@ and `load` can be used.
...
+#### Using host user secondary groups in container
+
+The Docker Compose ACAP is run by a non-root user on the device. This user is set
+up to be a member in a number of secondary groups as listed in the
+[manifest.json](https://github.com/AxisCommunications/docker-compose-acap/blob/rootless-preview/app/manifest.json#L6-L11)
+file. When running a container a user called `root`, (uid 0), belonging to group `root`, (gid 0)
+will be the default user inside the container. It will be mapped to the non-root user on
+the device, and the group will be mapped to the non-root users primary group.
+In order to get access inside the container to resources on the device that are group owned by any
+of the non-root users secondary groups these need to be added for the container user.
```
Comment (jojju) [inline README.md:343]: users'

File: README.md [hunk 2]
```diff
@@ -330,6 +331,31 @@ and `load` can be used.
...
+This can be done by using `group_add` in a docker-compose.yaml (`--group-add` if using Docker cli).
+Unfortunately, adding the names of the secondary groups are not supported, instead the *mapped* ids
+of the groups need to be used. At the moment of writing this the mappings are
+
+| device group | container group id |
+| ------------ | ------------------ |
+| datacache    | "1"                |
+| sdk          | "2"                |
+| sdk          | "2"                |
```
Comment (jojju) [inline README.md:352]: duplicate

**Review by jojju — COMMENTED** [review]
Great!

File: README.md (hunk 1 above)
Comment (killenheladagen) [reply README.md:343]: Perhaps a comman: "... by any of the non-root user's secondary groups, these need to.."

**Review by killenheladagen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (madelen-axis) [reply README.md:343]: The apostrophe is tricky
users' = possesion of several users
user's = user is
users = posesssion of one user

Comment (jojju) [reply README.md]: Right, I was going for possession of several users, but maybe "user's" would have been better now that I look at it again. "user's" means possession of one user. This is a tiny detail in any case.

**Pull request merged**